File: net.finah/API/Antwoord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace API
{
    class Antwoord
    {
        private int id;
        private int vraagId;
        private int rapportId;
        private int antwoord;
        private int extraAntwoord;
        private Boolean verzorger;

        public Antwoord(int id, int vraagId, int rapportId, int antwoord, int extraAntwoord, Boolean verzorger)
        {
            this.id = id;
            this.vraagId = vraagId;
            this.rapportId = rapportId;
            this.antwoord = antwoord;
            this.extraAntwoord = extraAntwoord;
            this.verzorger = verzorger;
        }

        public int setVraagId
        {
            get { return vraagId; }
            set { vraagId = value; }
        }

        public int setRapportId
        {
            get { return rapportId; }
            set { rapportId = value; }
        }

        public int setAntwoord
        {
            get { return antwoord; }
            set { antwoord = value; }
        }

        public int setExtraAntwoord
        {
            get { return extraAntwoord; }
            set { extraAntwoord = value; }
        }

        public Boolean setVerzorger
        {
            get { return verzorger; }
            set { verzorger = value; }
        }

        public String toString()
        {
            return id + ":" + vraagId + "," + rapportId + "," + antwoord + "," + extraAntwoord + " verzorger:" + verzorger;
        }
    }
}

File: finah-desktop-java/src/net/finah/API/Antwoord.java
package net.finah.API;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Antwoord {
	@JsonProperty("Id")
	private int id;

	@JsonProperty("Vraag_Id")
	private int vraag_Id;

	@JsonProperty("Rapport_Id")
	private int rapport_Id;

	@JsonProperty("AntwoordInt")
	private int antwoordInt;

	@JsonProperty("AntwoordExtra")
	private int antwoordExtra;

	@JsonProperty("Verzorger")
	private boolean verzorger;

	public void setId(int id) {
		this.id = id;
	}

	public void setVraag_Id(int vraag_Id) {
		this.vraag_Id = vraag_Id;
	}

	public void setRapport_Id(int rapport_Id) {
		this.rapport_Id = rapport_Id;
	}

	public void setAntwoordInt(int antwoordInt) {
		this.antwoordInt = antwoordInt;
	}

	public void setAntwoordExtra(int antwoordExtra) {
		this.antwoordExtra = antwoordExtra;
	}

	public void setVerzorger(boolean verzorger) {
		this.verzorger = verzorger;
	}

	public int getId() {
		return id;
	}

	public int getVraag_Id() {
		return vraag_Id;
	}

	public int getRapport_Id() {
		return rapport_Id;
	}

	public int getAntwoordInt() {
		return antwoordInt;
	}

	public int getAntwoordExtra() {
		return antwoordExtra;
	}

	public boolean getVerzorger() {
		return verzorger;
	}

	public String toString() {
		return id + ":" + vraag_Id + "," + rapport_Id + "," + antwoordInt + "," + antwoordExtra + " verzorger:" + verzorger;
	}
}
